gnomad_qc/v4/create_release/create_de_novo_release.py

```diff
@@ -223,13 +241,114 @@ def custom_an_select_globals(ht: hl.Table) -> Dict[str, hl.expr.Expression]:
     }
 
 
+def get_de_novo_config() -> Dict[str, dict]:
```

Comment:
We'll leave this as it is now?

Reply:
I'm not sure exactly what the question is

Reply:
Oh, I meant I just tested it and haven't looked at it carefully.

Reply:
Yeah that's fine, it will probably change, but I think we can put it in as is for now